pkg/cmd/kafka/list/list.go

```diff
@@ -87,31 +76,36 @@ func NewListCommand(f *factory.Factory) *cobra.Command {
 		},
 	}
 
-	flags := kafkaFlagutil.NewFlagSet(cmd, opts.localizer)
+	flags := kafkaFlagutil.NewFlagSet(cmd, opts.f.Localizer)
 
 	flags.AddOutput(&opts.outputFormat)
-	flags.IntVar(&opts.page, "page", int(cmdutil.ConvertPageValueToInt32(build.DefaultPageNumber)), opts.localizer.MustLocalize("kafka.list.flag.page"))
-	flags.IntVar(&opts.limit, "limit", 100, opts.localizer.MustLocalize("kafka.list.flag.limit"))
-	flags.StringVar(&opts.search, "search", "", opts.localizer.MustLocalize("kafka.list.flag.search"))
+	flags.IntVar(&opts.page, "page", int(cmdutil.ConvertPageValueToInt32(build.DefaultPageNumber)), opts.f.Localizer.MustLocalize("kafka.list.flag.page"))
+	flags.IntVar(&opts.limit, "limit", int(cmdutil.ConvertPageValueToInt32(build.DefaultPageSize)), opts.f.Localizer.MustLocalize("kafka.list.flag.limit"))
+	flags.StringVar(&opts.search, "search", "", opts.f.Localizer.MustLocalize("kafka.list.flag.search"))
+	flags.StringVar(&opts.clusterManagementApiUrl, "cluster-mgmt-api-url", "", f.Localizer.MustLocalize("dedicated.registerCluster.flag.clusterMgmtApiUrl.description"))
+	flags.StringVar(&opts.accessToken, "access-token", "", f.Localizer.MustLocalize("dedicated.registercluster.flag.accessToken.description"))
+
+	_ = flags.MarkHidden("cluster-mgmt-api-url")
+	_ = flags.MarkHidden("access-token")
```

Comment:
@jackdelahunt @dimakis let's consider adding these two flags for the:
- kafka create command
- all the dedicate command

Let's also hide them (just like https://github.com/redhat-developer/app-services-cli/blob/24ee8a625e9e46aa3946ac944ae4aa035ad8ad23/pkg/cmd/kafka/list/list.go#L88-L90) as they are simply there to enable smooth intergration with local environements e.g using kas-installer